src/fides/api/task/create_request_tasks.py

```diff
@@ -188,7 +188,7 @@ def base_task_data(
         # when executing the node, so we don't have to recalculate incoming
         # and outgoing edges.
         collection_representation = json.loads(
-            dataset_graph.nodes[node].collection.json()
+            dataset_graph.nodes[node].collection.model_dump_json(serialize_as_any=True)
```

Comment:
Another important place for DSR 3.0 - to save the collection on the RequestTask with duck typing so all the nested sub fields are persisted